### PR TITLE
[SDESK-1447] View/Edit mode for Events

### PR DESCRIPTION
**Pull request by nrvikas**

### CODE REVIEW

File: client/actions/events.js [hunk 1]
```diff
@@ -613,6 +613,16 @@ const closeAdvancedSearch = () => (
 )
 
 /**
+ * Opens the Event in preview/read-only mode
+ * @param {object} event - The Event ID to preview
+ * @return Promise
+ */
+const previewEvent = (event) => ({
+    type: EVENTS.ACTIONS.PREVIEW_EVENT,
+    payload: get(event, '_id', event || true),
```
Comment (vied12) [inline client/actions/events.js]: I find a bit confusing to mix `_id` and boolean here.
Can a user preview an event that has no `_id` (not saved event) ?

File: client/components/AddGeoLookupInput/index.jsx [hunk 2]
```diff
@@ -18,15 +18,17 @@ class GeoLookupInput extends React.Component {
         return (
             <Geolookup
                 placeholder='Address, City'
-                inputClassName="line-input"
-                buttonClassName='btn btn-default geolookup__button'
+                inputClassName={(this.props.readOnly ? 'disabledInput ' : '') + 'line-input'}
```
Comment (vied12) [inline client/components/AddGeoLookupInput/index.jsx]: It's totally fine. I just want to let you know that I introduced a `classNames` module that help to define classes like that.
https://github.com/JedWatson/classnames
Use it only if you like :)

File: client/actions/events.js (hunk 1 above)
Comment (nrvikas) [inline client/actions/events.js]: Good catch Edouard, I have changed it.

File: client/components/AddGeoLookupInput/index.jsx (hunk 2 above)
Comment (nrvikas) [inline client/components/AddGeoLookupInput/index.jsx]: Sure, that's a better approach. Tks.

**Pull request merged**